Source/EventFlow.ReadStores.Elasticsearch/Extensions/EventFlowOptionsExtensions.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using Elasticsearch.Net.ConnectionPool;
+using EventFlow.Aggregates;
+using EventFlow.Configuration.Registrations;
+using EventFlow.Logs;
+using Nest;
+
+namespace EventFlow.ReadStores.Elasticsearch.Extensions
+{
+    public static class EventFlowOptionsExtensions
+    {
+        public static EventFlowOptions UseElasticsearchReadModel<TAggregate, TReadModel>(this EventFlowOptions eventFlowOptions, params string[] indicesToFeed)
```

Comment:
@tabro What if the indexes change at run-time? I don't have any solution, but given the task we did last week, we may need to put read models in more than one index.